org.sugarlabs.MusicBlocks.yaml

```diff
@@ -0,0 +1,115 @@
+app-id: org.sugarlabs.Musicblocks
```

Comment:
The error you see is because the `app-id` doesn't match. It should be `org.sugarlabs.MusicBlocks` capital B. This shouldn't have worked either locally.

Reply:
I see, thanks! I'm not sure why it worked locally – must have been something left over from previous builds.